
start_pose: None
target_pose: None
planner_select: Cano_etal
planner: all
mode: tpe
avg_runs: 1
max_trials: 30
max_runtime: None
max_plantime: 3